app/css/views/themes.css

```diff
@@ -42,4 +42,5 @@
   margin: 0 0 0 1rem;
   padding: 0;
   right: 0;
+  font-size: 0;
```

Comment:
> And yes, we need to work a little on accessibility, but I don't have enough knowledge in that field.

Ah, np, I think we can plan accessibility week/days if we get approval for the next foxbox stage :)